docs/src/api/class-playwright.md

```diff
@@ -76,6 +76,12 @@ This object can be used to launch or connect to Chromium, returning instances of
 
 Returns a dictionary of devices to be used with [`method: Browser.newContext`] or [`method: Browser.newPage`].
 
+## property: Playwright.devices
```

Comment:
As discussed in the meeting today, let's drop this method from C# API altogether.